packages/vite/src/node/server/__tests__/pluginContainer.spec.ts

```diff
@@ -226,7 +226,16 @@ async function getDevEnvironment(
   // @ts-expect-error This plugin requires a ViteDevServer instance.
   config.plugins = config.plugins.filter((p) => !p.name.includes('pre-alias'))
 
-  const environment = new DevEnvironment('client', config, { hot: false })
+  const environment = new DevEnvironment('client', config, {
+    hot: false,
+    transport: {
```

Comment:
`transport` is not required anymore

Reply:
Fixed in 878849c8f47339fbb5030c42808da4af253d4489